### ISSUE TITLE
pan view MSA: allow columns to multi-select the gene families present at a given alignment position

### ISSUE THREAD

**Issue by adf-ncgr** [legumeinfo/lis_context_viewer#84]
sometimes, you'll find that tracks in an alignment differ at a certain position with respect to the gene family assigned for the gene in that position (ie the "color" alternates between a few choices). The question that naturally occurs is whether these different families are themselves related in some obvious way (e.g. the assigned functional descriptors are similar, or there is commonality among the protein domain content of the two families). Currently, it is possible to address the question by mousing over the families one-by-one in the legend and following links; Having some UI element to represent the family composition at a position in an alignment and use it to get directly to the set of gene families represented would be a nice-to-have.

**Comment by alancleary** [comment]
Indeed, that would be a nice feature. It may be non-trivial to implement though. Currently we're using a single micro-synteny viewer to display all the tracks and simply sorting by name to cause tracks in the same group to be displayed together. So if we want to add group specific column interaction then we need to decide whether to put each group into its own viewer or make the viewer explicitly aware of track groupings.

**Comment by adf-ncgr** [comment]
agreed that it is nontrivial; I will icebox it for now. can imagine revisiting it when we have transitioned our "groups" to "metadata attributes" and allow not only filtering by metadata but also grouping. Likely this will not occur until you are Dr. Alan Cleary...
